openml/datasets/functions.py

```diff
@@ -424,6 +424,10 @@ def get_dataset(
                 raise
 
         arff_file = _get_dataset_arff(description) if download_data else None
+        if "oml:minio_url" in description and download_data:
```

Comment:
Could you please explain why we potentially download both files? I guess this is easier to handle at the moment?

Reply:
The main reason is that the test server currently always returns a `minio_url` entry, regardless of whether or not the parquet file actually exists. I suppose you could turn it around and only download the arff file of the parquet file needed to download. 
Some people also might still be interested in having the arff file (for now), and we would not have a public API for downloading that file, so in the interest of merging this before a PyPI release, I figured keeping that behavior for now is better.

Reply:
That makes sense. Do you think we should open an issue to track the move from arff to parquet?

Reply:
Seems like a good idea: #1032.